tests/fixtures/docker_cluster_fixtures.py

```diff
@@ -379,6 +379,7 @@ def docker_cluster_pk_ssh(request, test_org_rns_folder):
         ],
         working_dir=None,
         name="default_env",
+        env_vars={"RH_LOG_LEVEL": os.getenv("RH_LOG_LEVEL") or TESTING_LOG_LEVEL},
```

Comment:
oh if you set this locally will it propagate server side?

Reply:
yea as part of the restart flow / `env.to(cluster)`